jax_galsim/__init__.py

```diff
@@ -21,10 +21,20 @@
     GalSimWarning,
 )
 
+# Bessel
+from .bessel import J0
+
 # Basic building blocks
 from .bounds import Bounds, BoundsD, BoundsI
+from .box import Box, Pixel
```

Comment:
I think this __init_.py was an artifact of git manipulation that I do not master. I first do the repo clone and make a moffat branch from the main but on 2nd August @ismael-mendoza told me that " I apologize, I should have mentioned that you should do this branch based of the merge_conv ". So then it may have happened some git pb.